activerecord/lib/active_record/attribute_methods.rb

```diff
@@ -288,6 +288,15 @@ def attribute_for_inspect(attr_name)
       format_for_inspect(attr_name, value)
     end
 
+    def attribute_for_pretty_print(attr_name)
+      value = _read_attribute(attr_name)
+      if value.is_a?(String) && value.length > 50
+        "#{value[0, 50]}..."
```

Comment:
You can use a single [ellipsis character](https://www.compart.com/en/unicode/U+2026):
```suggestion
        "#{value[0, 50]}…"
```